ambari-server/src/main/resources/stack-hooks/after-INSTALL/scripts/hook.py

```diff
@@ -29,9 +29,9 @@ def hook(self, env):
 
     env.set_params(params)
     setup_stack_symlinks(self.stroutfile)
```

Comment:
We don't need to call stack-select anymore for mpacks anymore.

Reply:
I have merged this change to unblock the deployments. You can take care of this in your next commit.